recipes/pygadm/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pygadm" %}
+{% set version = "0.0.0" %}
```

Comment:
Is this correct?

Reply:
not anymore ;-) let me do a quick update